### DISCUSSION TITLE
GitHub Copilot Monthly Digest 🗞️

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#64055] [Copilot News and Announcements]
👋 Greetings and welcome to the first edition of Copilot Monthly Roundup. As we navigate the ever-evolving landscape of AI advancements in Copilot, we're excited to present a consolidated source of information containing the latest from Copilot 🤖. Our aim is to keep you well-informed about pivotal Copilot developments, including new launches, enhanced features, trends, and more! 

🔢 Let’s Go!

### 1️⃣ Copilot for Business Chat 💼 

<img width="1780" alt="Screenshot 2023-08-17 at 3 02 57 PM" src="https://github.com/community/community/assets/95834130/e00eb768-ebe1-4710-9282-3f826c8eb34f">

GitHub Copilot for Business users can now access a limited GitHub Copilot Chat beta, bringing the power of conversational coding right to the IDE.

It is more than a chat window. It recognizes what code a developer has typed, what error messages are shown, and it’s deeply embedded into the IDE. Key features include:

- Personalized assistance
- Real-time guidance
- Fixing security issues
- Code analysis
- Simple troubleshooting

### 2️⃣  Copilot Trust Centre 🆕 

https://github.com/community/community/assets/95834130/4d23e3c0-4b08-4929-81c4-2f4da7bc8675

This past month we enabled developers and organizations to maximize their potential by prioritizing security, privacy, compliance, and transparency as we develop and iterate on GitHub Copilot.

Find out more 🔍  in the [GitHub Copilot Trust Centre](https://resources.github.com/copilot-trust-center/)

### 3️⃣ [Try out code referencing [Private Beta]!](https://github.blog/2023-08-03-introducing-code-referencing-for-github-copilot/)

Make more informed decisions about the code you use. In the rare case where a GitHub Copilot suggestion matches public code, this update will show a list of repositories where that code appears and their licenses. Explore more!

### 4️⃣   [🤖 Automate GitHub Copilot access for your organization with User Management API (beta)](https://docs.github.com/en/rest/copilot/copilot-for-business?apiVersion=2022-11-28)

Use the REST API to manage the GitHub Copilot for Business subscription for your organization. Check out the [documentation](https://docs.github.com/en/rest/copilot/copilot-for-business?apiVersion=2022-11-28#get-copilot-for-business-seat-information-and-settings-for-an-organization) for more details. ⚡ 

---
### 🧢  An easier way manage your Copilot for Individuals trial and plan 😮 

<img width="1326" alt="Screenshot 2023-08-14 at 11 04 10 AM" src="https://github.com/community/community/assets/95834130/b1e18cbd-e7dc-4b6d-81f0-3f5e37491440">

We've heard confusion from users on how to switch between monthly and annual billing. We want you to feel fully in control of your GitHub Copilot plan so we've updated the **[Plans and Usage](https://github.com/settings/billing)** page to make it easier to swap between your plan options. Just head down to the Copilot plan section and hit the Manage subscription button to see your options. 

### 💡  [Select languages setting within IntelliJ](https://github.blog/changelog/2023-08-11-github-copilot-august-10th-update/#select-languages-setting-within-intellij)

We’ve replaced the previous `disabledLanguages` configuration with a new, more flexible `languageAllowList` configuration. This change allows enabling or disabling all languages at once using the wildcard. Get more done in less time 🚀 

### 🧑‍💻  [Your Support experience is now powered ⚡ by Copilot - Copilot in GitHub Support (Alpha)](https://github.blog/changelog/2023-08-04-copilot-in-github-support-alpha/)

We’ve released Copilot in GitHub Support, a faster way to find answers to your GitHub related questions! Copilot in GitHub Support is an AI-powered assistant that answers questions based on our official GitHub Enterprise Cloud documentation.

Initially, we’re offering the Copilot experience to a limited number of randomly selected GitHub Enterprise Cloud customers. We hope to continue rolling out the experience to a wider audience over the coming months. 🚀

---
### 1️⃣ [Troubleshooting ⚙️  network errors for GitHub Copilot and view logs in your IDE](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-network-errors-for-github-copilot)

View logs to troubleshoot GitHub Copilot-related errors in your IDE and resolve common errors related to proxies and custom certificates. 👐 

### 2️⃣ [Know how’s of GitHub Copilot Chat and its use cases](https://docs.github.com/en/copilot/github-copilot-chat/about-github-copilot-chat)

[GitHub Copilot Chat](https://github.blog/2023-07-20-github-copilot-chat-beta-now-available-for-every-organization/) can help you by providing answers to coding related questions directly within a supported IDE. We created a new documentation to make you use Copilot Chat effectively - from prerequisites, to asking the first question, and getting your answers. ✅ 

### 3️⃣ [Troubleshooting authentication issues with GitHub Copilot Chat](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-authentication-issues-with-github-copilot-chat)

We’ve documented the process and  diagnosed common authentication issues with GitHub Copilot Chat in VSCode and Visual Studio. ⌨️ 

---
## Top learning content posted:

- [Tips for building software with generative AI](https://github.com/orgs/community/discussions/56349)
- [Step-by-Step: How to Setup Copilot Chat in VS Code](https://github.com/orgs/community/discussions/58756)
- [📅 Month of Learning 📚 - Copilot 🤖 Community Check-in](https://github.com/orgs/community/discussions/59644)
- [Using GitHub Copilot for MLOps, TDD, & more](https://github.com/orgs/community/discussions/57344)

## Trending 📊 in the Community 

- Resolved - [GitHub Copilot plugin is not compatible with IntelliJ IDEA 2022.1 and Java 11](https://github.com/orgs/community/discussions/60836)
- Bug Identified - [Visual Studio 2022 “GitHub CoPilot has not been started”](https://github.com/orgs/community/discussions/61137)
- Resolved - [Copilot for Android Studio. Can not login to GitHub](https://github.com/orgs/community/discussions/60859)
- New Feature 🚢 - [Inquiry Regarding GitHub Copilot Metrics and Productivity Insights](https://github.com/orgs/community/discussions/59506) 

### Resources to bookmark🔖

- [GitHub Copilot X FAQs](https://github.com/features/preview/copilot-x#:~:text=Frequently%20asked%C2%A0questions)
- [Copilot on GitHub Blogs](https://github.blog/tag/github-copilot/)
- [🤖 Copilot for Business User Management API (beta)](https://github.com/orgs/community/discussions/60780)
- [🤖 Introducing GitHub Copilot Chat beta for businesses 💼](https://github.com/orgs/community/discussions/61415)
- [GitHub Copilot Nightly replaced by Copilot pre-release extension](https://github.com/orgs/community/discussions/59964)

To learn more about Copilot and take full advantage of all of its power, visit our [YouTube Copilot playlist](https://www.youtube.com/playlist?list=PL0lo9MOBetEHEHi9h0k_lPn0XZdEeYZDS). To sign up, see our [Copilot features](https://github.com/features/copilot) page. 

Got questions or feedback on anything Copilot? Post them in the [Community on Discussions](https://github.com/orgs/community/discussions/categories/copilot). Happy Copiloting! Happy Coding! 🚀

**Comment by Mahmood3214** [comment]
Very good

**Comment by Binary-Ninja-007** [comment]
Good.
